### ISSUE TITLE
[bug] Weird pull vehicle from garage glitch

### ISSUE THREAD

**Issue by Bob-Murphy** [official-antistasi-community/A3-Antistasi#457]
https://youtu.be/-0YBpoYuYok

> [23:38] Burbonizer: this is what is happening
> [23:38] Burbonizer: this is the first time i have really run Antistasi and this bug i frustrating
> [00:19] Bob Murphy: Spoffy Please take a look at the video above
> Burbonizer Could you please send the corresponding RPT via DM?
> [00:21] Burbonizer: i will have to run the same test again asi dont have logs running
> [00:23] poweredbypot: what the actual fuck
> [00:23] Burbonizer: and for the test i will be placing a vehicle down via zeus to save me going to hunt for one
> [00:23] poweredbypot: you can also do player setVariable ["moneyX",1000,true] and global execute from console
> [00:23] poweredbypot: so you can just buy one
> [00:24] Bob Murphy: Please don't do the vehicle spawning with the next test. Go into Zeus and just move a spawned vehicle over to base.
> [00:35] Bob Murphy:
> 23:25:24 "BURBONIZER/BIS_fnc_log: [module] Ares_fnc_ZeusVisibility (1003.01 ms)"
> 23:25:46 "BURBONIZER/BIS_fnc_log: [BIS_fnc_curatorObjectEdited] [42,[L Zeus:1,21983f50080# 1696136: hatchback_01_f.p3d REMOTE]]"
> 23:25:54 "BURBONIZER/BIS_fnc_log: [module] Ares_fnc_PlayerTeleport (1007 ms)"
> 23:26:29 Cannot create non-ai vehicle ,
> hmmmm
> [00:37] Bob Murphy: have you ever tried another car? I just wonder because the full class normally is 
> C_Hatchback_01_F
>  and the "C_" is missing. Of course it could be that it's for some reason not stated completely.
> [00:41] Spoffy: Huh. Neat bug.
> [00:41] Bob Murphy: Spoffy the four lines i posted above is all what is in the rpt
> [00:42] Spoffy: Will look into it.
> [00:42] Spoffy: But can you confirm if it happens with other vehicles Burbonizer ?
> [00:42] Spoffy: Can you open me an issue actually Bob Murphy so I don't forget? :)
> [00:42] Bob Murphy: aye
> [00:43] Burbonizer: i tried it with the hatchback and a truck, open and covered.
> [00:43] Burbonizer: just wont spawn
> [00:43] Burbonizer: but....
> [00:43] Burbonizer: if i try the same thing in the editor it seems to work

**Comment by Bob-Murphy** [comment]
Second test

> 23:58:54 "BURBONIZER/BIS_fnc_log: [BIS_fnc_curatorObjectEdited] [42,[L Zeus:1,28fbdbe6040# 1696090: truck_02_transport_f.p3d REMOTE]]"
> 23:58:58 Client: Network message 1048 (update info 364eb428) is pending
> 23:59:03 Client: Network message 1147 (update info 364eb428) is pending
> 23:59:08 "BURBONIZER/BIS_fnc_log: [BIS_fnc_curatorObjectEdited] [42,[L Zeus:1,28fbdbe6040# 1696090: truck_02_transport_f.p3d REMOTE]]"
> 23:59:08 Client: Network message 125d (update info 691335a8) is pending
> 23:59:11 "BURBONIZER/BIS_fnc_log: [module] Ares_fnc_PlayerTeleport (1008 ms)"
> 23:59:12 a3\characters_f\heads\glasses\g_shades_black.p3d: No geometry and no visual shape
> 23:59:13 Client: Network message 13ad (update info 691335a8) is pending
> 23:59:15 "BURBONIZER/BIS_fnc_log: [module] Ares_fnc_ZeusVisibility (1007 ms)"
> 23:59:18 Client: Network message 14d7 (update info 691335a8) is pending
> 23:59:23 Client: Network message 1636 (update info 691335a8) is pending
> 23:59:28 Client: Network message 171c (update info 691335a8) is pending
> 23:59:33 Client: Network message 17f2 (update info 691335a8) is pending
> 23:59:38 Client: Network message 1908 (update info 282e2828) is pending
> 23:59:43 Client: Network message 1a0d (update info 282e2828) is pending
> 23:59:44 Cannot create non-ai vehicle ,

**Comment by nirousseau** [comment]
i am also facing the same issue (maybe it is the very last update that broke things ?)

**Comment by hypertexthero** [comment]
Also seeing what I think may be the same issue, but it happens when attempting to purchase a quadbike in the beginning of the [Malden Beta](https://steamcommunity.com/sharedfiles/filedetails/?id=1850103246) campaign. I don’t see the vehicle at all when the message "hover cursor to place vehicle" message appear, no matter where I point the cursor.

**Comment by Spoffy** [comment]
Thanks for all the reports guys, keep them coming if there's any more to add! I'll try and get this fixed in the next release.

**Comment by Burbonizer** [comment]
Do you think this could be due to the latest Arma release Spoffy? I have ran antistasi before, just to try it, and everything worked fine. This was about 1 month ago.

**Comment by Spoffy** [comment]
Could be, I've never seen this bug before, and I'm having difficulty replicating it. Are you running on a dedicated server or LAN Host? And are you running 2.1 or 2.2?

**Comment by Spoffy** [comment]
@hypertexthero @nirousseau Can you please link me to the version of the mod you're using?

**Comment by nirousseau** [comment]
Yes,

Hosted on a Linux Dedicated Servver, with latest Arma 3 Update with :
- https://github.com/CBATeam/CBA_A3/releases/tag/v3.12.2.190909
- https://github.com/acemod/ACE3/releases/tag/v3.12.6
- https://github.com/DerZade/BackpackOnChest/releases/tag/1.2.0

The mission is  **A3-Antistasi-2_1_2-Altis.Altis.pbo** download from the last github release.

**Comment by Spoffy** [comment]
Fantastic, thanks!

**Comment by Spoffy** [comment]
Found the cause of this, thanks to @Burbonizer for all their help in allowing me access to their server to replicate the issue.

The cause is that essentially, createVehicleLocal doesn't work in Multiplayer, unless a specific parameter is set (that was enabled by default up until 1.96).

Bohemia are testing a hotfix that restores the old behaviour:
https://forums.bohemia.net/forums/topic/183855-release-candidate-branch-announcements/?page=2&tab=comments#comment-3384782

Original changelog:
`Changed: createVehicleLocal was restricted in multiplayer - enterable vehicles, shots, explosions and similar are no longer possible to spawn (the old behavior can be turned on via "unsafeCVL=1" in the description.ext or server's config) `

TL;DR Sit tight, wait for the hotfix. Bohemia fucked up.

**Comment by nirousseau** [comment]
top ! :)
